addons/rivet/rivet_global.gd

```diff
@@ -7,15 +7,15 @@ extends Node
 ## @experimental
 
 const _api = preload("api/rivet_api.gd")
-const _cli = preload("devtools/rivet_cli.gd")
-
-var cli := _cli.new()
 
 var cloud_token: String
 var namespace_token: String 
 var game_id: String
 var api_endpoint: String
 
+# This variable is only accessible from editor's scripts, please do not use it in your game.
+var cli
```

Comment:
Would it make sense to name this `_cli` or `_internal_cli` or something similar?

Can this have a type?